lib/season.rb

```diff
@@ -1,10 +1,70 @@
 class Season
   def initialize(game_team_data, game_data, team_data)
+
     @game_team_data = game_team_data
     @game_data = game_data
     @team_data = team_data
+    @games_data = games_data
   end
 
+
+  def most_tackles(season)
```

Comment:
We could break these into helper methods or branch them off into modules.

Stuff for tomorrows refactoring